less/index.less

```
@@ -57,3 +58,256 @@ html, body {
     height: 100%;
 }
 
+html {
+	background-image: url(https://i.ibb.co/XZhq4dv/circle-bg.png);
+	background-size: 1000px;
+	
+}
+
+// MY VARIABLES //
+@h2-fonts: 2rem;
+@p-fonts: 1rem;
+
+// MY MIXINS //
+
+.h2-color {
+	color: #4d4d4d;
+	font-weight: bold;
+	font-family: 'Gentium Basic', serif;
+	font-style: italic;
+	
+}
+
+.p-style {
+	color: #4d4d4d;
+	padding-top: 2%;
+	padding-bottom: 2%;
+	font-family: 'Roboto', sans-serif;
+	font-size: 1rem;
+	line-height: 35px;
+	font-weight: bold;
+}
+
+.font-family {
+	font-family: 'Roboto', sans-serif;
+}
+
+.main-color (
+	color: #dcffff;
+)
```

Are you using a linter with VS Code? They can be super helpful for catching errors like this. Maybe your VS Code wasn't yelling at you to use brackets instead of parenthesis.  Here's one that checks preprocessor syntax:
https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint

```suggestion
.main-color {
	color: #dcffff;
}
```

---

Actually, when I fixed the syntax of this it broke the layout. Which tells me you didn't actually intend to use it. You can probably get rid of it.